src/locale/en.json

```
@@ -118,8 +121,11 @@
         "C4": "C4 - landfill",
         "D": "D - reuse"
       },
-      "totalCO2Equivalents": "CO2 emissions from product (so far)",
-      "years": "Years"
+      "timeunit": {
+        "month": "{count, plural, =0 {less than a month} =1 {one month} other {# months}}",
+        "year": "{count, plural, =0 {less than a year} =1 {one year} other {# years}}"
```

Same as above to keep it consistent. 
```suggestion
        "month": "{count, plural, =0 {less than a month} =1 {# month} other {# months}}",
        "year": "{count, plural, =0 {less than a year} =1 {# year} other {# years}}"
```